src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/CreditCardMapper.php

```diff
@@ -83,10 +84,14 @@ protected function getUrlToComputop($merchantId, $data, $length)
             ComputopApiConfig::DATA => $data,
             ComputopApiConfig::LENGTH => $length,
             ComputopApiConfig::URL_BACK => $this->getAbsoluteUrl(
-                    $this->application->path($this->config->getCallbackFailureRedirectPath())
-                )
+                $this->application->path($this->config->getCallbackFailureRedirectPath())
+            ),
         ];
 
+        if ($this->config->getCreditCardTemplateEnabled()) {
+            $queryData[ComputopApiConfig::TEMPLATE] = $merchantId;
```

Comment:
Why is it `merchantId` here?
I would expect template set somewhere to be used here.

Reply:
From computop documentation:
* Always use your MerchantID in the names of the files.
* example: https://www.computop-paygate.com/payssl.aspx?MerchantID=YourMID&Len=123&Data=AGSDJ…ASDF&template=YourMerchantID

Reply:
thanks, now it's clear